modules/ols-granting-access-to-individual-users.adoc

```diff
@@ -0,0 +1,26 @@
+// This module is used in the following assemblies:
+// configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ols-granting-access-to-individual-users_{context}"]
+= Granting access to an individual user
+
+This procedure explains how to grant access to an individual user.
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in as a user with the ability to grant permissions.
+
+* You have deployed the {ols-long} service.
+
+* You have access to the {ocp-short-name} CLI (oc).
+
+.Procedure
+
+. Run the following command at the command line:
++
+[source,terminal]
+----
+oc adm policy add-cluster-role-to-user \
+lightspeed-operator-query-access <user_name>
+----
```

Comment:
at some point in the future we should add the YAML for this, too.

Reply:
Created [OLS-866](https://issues.redhat.com//browse/OLS-866) to capture the action to add the file later on.